kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -73,26 +73,73 @@ def exec_kompile(
     backend: KompileBackend,
     main_file: Path,
     emit_json: bool,
+    kompile_mode: str,
```

Comment:
```suggestion
    kompile_mode: Union[str, KompileMode],
```

where

```python3
class KompileMode(Enum):
    NODE = 'node'
    STANDALONE = 'standalone'
```

Then you can process it like this:

```python3
kompile_mode = KompileMode(kompile_mode)
```

Reply:
I ended up using an Enum directly, have a look.